--- cfns.gperf	2013-01-10 20:38:27.000000000 +0000
+++ /home/yongqian/Downloads/cfns.gperf	2026-01-14 10:03:50.589515891 +0000
@@ -16,14 +16,6 @@
 You should have received a copy of the GNU General Public License
 along with GCC; see the file COPYING3.  If not see
 <http://www.gnu.org/licenses/>.  */
-#ifdef __GNUC__
-__inline
-#endif
-static unsigned int hash (const char *, unsigned int);
-#ifdef __GNUC__
-__inline
-#endif
-const char * libc_name_p (const char *, unsigned int);
 %}
 %%
 # The standard C library functions, for feeding to gperf; the result is used
