--- except.c	2013-01-10 20:38:27.000000000 +0000
+++ /home/yongqian/Downloads/except.c	2026-01-14 10:04:46.684841841 +0000
@@ -1022,7 +1022,7 @@
      unless the system headers are playing rename tricks, and if
      they are, we don't want to be confused by them.  */
   id = DECL_NAME (fn);
-  return !!libc_name_p (IDENTIFIER_POINTER (id), IDENTIFIER_LENGTH (id));
+  return !!libc_name::libc_name_p (IDENTIFIER_POINTER (id), IDENTIFIER_LENGTH (id));
 }
 
 /* Returns nonzero if an exception of type FROM will be caught by a
